### PR TITLE
Migrate documentation from `README.md` to Sphinx

### PR DESCRIPTION
**Pull request by navaneethkulal8**
Create Sphinx documentation #10

### CODE REVIEW

**Review by ConorMacBride — COMMENTED** [review]
Thanks @navaneethkulal8! Just a few updates

File: docs/source/index.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+   build-and-publish-python-package
+   build-and-publish-pure-python-package
+
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```
Comment (ConorMacBride) [inline docs/source/index.rst:22]:
```suggestion
```

**Comment by navaneethkulal8** [comment]
Hello @ConorMacBride the updates which you gave, should i change that and update or is it all ready changed by you and merged?

**Comment by navaneethkulal8** [comment]
And is the issue solved?

**Comment by ConorMacBride** [comment]
Yes. Once you commit the suggestions I will merge the PR, and then I'll publish it as a website!

**Pull request merged**